frame/merkle-mountain-range/src/mock.rs

```diff
@@ -91,14 +94,14 @@ impl LeafData {
 	}
 }
 
-thread_local! {
-	pub static LEAF_DATA: RefCell<LeafData> = RefCell::new(Default::default());
+parameter_types! {
+	pub static LeafDataTestValue: LeafData = Default::default();
```

Comment:
Again, don't see why you renamed this.

Reply:
I can't name it `LeafData` because there's already a struct named `LeafData`